docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -15,69 +15,53 @@ authenticate to external services.
 
 - (!docs/pages/includes/tctl.mdx!)
 - If you're new to SAML, consider reviewing our [SAML Identity Provider
-Reference](./saml-reference.mdx) before proceeding.
+  Reference](./saml-reference.mdx) before proceeding.
+- User with permission to create Service Provider resource. The preset `editor` role has this permission.
```

Comment:
We should explain what a Service Provide resource is

Reply:
The glossary for SAML IdP is published on this page https://goteleport.com/docs/access-controls/idps/saml-reference/, which also covers service provider, and the reference page is linked just before this line.